filter_plugins/oo_filters.py

```diff
@@ -881,14 +881,13 @@ def oo_image_tag_to_rpm_version(version, include_dash=False):
         """
         if not isinstance(version, basestring):
             raise errors.AnsibleFilterError("|failed expects a string or unicode")
-        # TODO: Do we need to make this actually convert v1.2.0-rc1 into 1.2.0-0.rc1
-        # We'd need to be really strict about how we build the RPM Version+Release
```

Comment:
Man, the new origin tagging is going to throw a wrench into this...  Maybe we can tackle that in packaging, where we if upstream specifies something that would look like a 'release' we use it as the 'release' in the subsequent packaging. 

I can't think of a reason that we would need to track a separate release than upstream, and if so, maybe we can just tack on a second "release", so 1.2.0-rc1-0 ?  I must say that I'm pretty ignorant of t he edge cases of RPM versioning, though.

@sdodson @tdawson thoughts?